synapse/storage/profile.py

```diff
@@ -55,3 +57,99 @@ def set_profile_avatar_url(self, user_localpart, new_avatar_url):
             updatevalues={"avatar_url": new_avatar_url},
             desc="set_profile_avatar_url",
         )
+
+    def get_from_remote_profile_cache(self, user_id):
```

Comment:
Is this actually called anywhere? Or is the cache write-only at the moment?

Reply:
This PR is TBC...